homeassistant/components/emoncms_history/__init__.py

```diff
@@ -86,15 +86,13 @@ def update_emoncms(time):
                 continue
 
         if payload_dict:
-            payload = "{{{}}}".format(
-                ",".join(f"{key}:{val}" for key, val in payload_dict.items())
-            )
+            payload = ",".join(f"{key}:{val}" for key, val in payload_dict.items())
 
             send_data(
                 conf.get(CONF_URL),
                 conf.get(CONF_API_KEY),
                 str(conf.get(CONF_INPUTNODE)),
-                payload,
+                f"{{{payload}}}",
```

Comment:
I think this should be `{payload}`, not `{{{payload}}}`.

Reply:
Here the `{}` are being added as have been removed from a previous statement to make the statement simpler.